test/apiv2/20-containers.at

```diff
@@ -263,3 +263,12 @@ t GET containers/json 200 \
   .[0].Ports[0].Type="tcp"
 
 podman stop bar
+
+# Test CPU limit (NanoCPUs)
+t POST containers/create '"Image":"'$IMAGE'","HostConfig":{"NanpCpus":500000}' 201 \
```

Comment:
typo in `NanpCpus`?

Reply:
Good catch... I was trying to figure out why this wasn't being set.